src/AppBundle/Controller/Org/ClientContactController.php

```diff
@@ -68,7 +68,7 @@ public function editAction(Request $request, $id)
 
         $this->denyAccessUnlessGranted('edit-client-contact', $client, 'Access denied');
 
-        $form = $this->createForm(new FormDir\Org\ClientContactType($this->get('translator')), $clientContact);
+        $form = $this->createForm(FormDir\Org\ClientContactType::class, $clientContact);
```

Comment:
q: Does this still form translate ok?

Reply:
yes. the translator wasn't used inside (previous refactor I think). It's generally not needed inside forms, as symfony translates choice values too (in symfony 3, they are now the keys, no the valeus of the "choices" array)